### DISCUSSION TITLE
Buffer of 2 bytes

### DISCUSSION THREAD

**Discussion by volldasgute** [pail23/stiebel_eltron_isg_component#33] [Q&A]
Config entry 'Stiebel Eltron ISG' for stiebel_eltron_isg integration not ready yet: unpack requires a buffer of 2 bytes; Retrying in background

Hello folk,

thats the Warning i got after installation... could anybody help?

Thanks
Frank

**Comment by pail23** [comment]
It looks like home assistant did not receive enough data from the heat pump. Which version of the integration do you use? What is your heat pump model?

**Comment by pail23** [comment]
The same issue was reportet in issue #40. Should be solved in V0.12.1

**Comment by volldasgute** [comment]
Thanks a lot. Now it works!
so now i could check out SG Ready.
Could you Tell me which Mode i have to configure in the ISG Web Menü?
ISG or Modus?

Kind regards Frank
